test/lib/aws-server-stubs/ai-server/index.js

```diff
@@ -119,6 +119,12 @@ function handler(req, res) {
         break
       }
 
+      case 'meta.llama3-8b-instruct-v1:0':
+      case 'meta.llama3-70b-instruct-v1:0': {
+        response = responses.llama3.get(payload.prompt)
```

Comment:
since llama3 responses are identical you don't really need mocks. you could just return the llama2 data